idxmgmt/ilm/config.go

```diff
@@ -86,11 +87,11 @@ func NewConfig(info beat.Info, cfg *libcommon.Config) (Config, error) {
 	// replace variable rollover_alias parts with beat information if available
 	// otherwise fail as the full alias needs to be known during setup.
 	for et, m := range config.Setup.Mappings {
-		rolloverAlias, err := applyStaticFmtstr(info, m.RolloverAlias)
+		idx, err := applyStaticFmtstr(info, m.Index)
 		if err != nil {
-			return Config{}, errors.Wrap(err, "variable part of rollover_alias cannot be resolved")
+			return Config{}, errors.Wrap(err, "variable part of index suffix cannot be resolved")
```

Comment:
`variable part of index suffix` ==> isn't this just the "index"?

Reply:
Technically yes, but the everything except the suffix is fixed by us. The error message reuses the term _index suffix_  as this is what the users can configure and are used to from the `apm-server.yml`.